src/docs/webpackEntry.jsx

```diff
@@ -0,0 +1,22 @@
+import {hot} from 'react-hot-loader/root';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '../main.scss';
+import './_sass/main.scss';
+import Entry from './pages/interactive';
+import {sections} from './page-components';
+
+const requireAll = r => {
+  r.keys().forEach(r);
+};
+
+requireAll(require.context('../images/', true, /\.svg$/));
```

Comment:
What does it do? :D

Reply:
it's a weird way in webpack to load assets recursively, more details here: https://github.com/webpack/docs/wiki/context#context-module-api